src/mxmake/topics/core/base.mk

```
@@ -72,6 +72,6 @@ MXMAKE_FOLDER?=.mxmake
 # Sentinel files
 SENTINEL_FOLDER?=$(MXMAKE_FOLDER)/sentinels
 SENTINEL?=$(SENTINEL_FOLDER)/about.txt
-$(SENTINEL):
+$(SENTINEL): Makefile
```

Thus this works, I would generalize it:
https://www.gnu.org/software/make/manual/html_node/Special-Variables.html -> `MAKEFILE_LIST`

```suggestion
$(SENTINEL): $(firstword $(MAKEFILE_LIST))
```